frappe/templates/web.html

```diff
@@ -4,14 +4,13 @@
 {% block content %}
 <div class="page-container" id="page-{{ name or page_name }}" data-path="{{ pathname }}"
 	{% if page_or_generator=="Generator" %}data-doctype="{{ doctype }}"{% endif %}>
-		<div class="page-content {% if not show_sidebar %}page-max-width{% endif %}">
 			<div class="row {% if show_sidebar %}vert-line{% endif %}">
 				{% if show_sidebar %}
 					<div class="col-sm-3 sidebar-block">
 						{% include "templates/includes/web_sidebar.html" %}
 					</div>
 				{% endif %}
-				<div class="{% if show_sidebar %}content-block col-sm-9{% else %} content-block col-sm-12 {% endif %}">
+				<div class="{% if show_sidebar %}page-content col-sm-9{% else %} page-content col-sm-12 {% endif %}">
```

Comment:
But then you will have to fix our other websites that apply CSS on
content-block. Check all the repos under frappe.

On Thu, May 5, 2016 at 10:35 PM, KanchanChauhan <notifications@github.com>
wrote:

> In frappe/templates/web.html
> <https://github.com/frappe/frappe/pull/1744#discussion_r62218261>:
>
> >  			<div class="row {% if show_sidebar %}vert-line{% endif %}">
> >  				{% if show_sidebar %}
> >  					<div class="col-sm-3 sidebar-block">
> >  						{% include "templates/includes/web_sidebar.html" %}
> >  					</div>
> >  				{% endif %}
> > -				<div class="{% if show_sidebar %}content-block col-sm-9{% else %} content-block col-sm-12 {% endif %}">
> > +				<div class="{% if show_sidebar %}page-content col-sm-9{% else %} page-content col-sm-12 {% endif %}">
>
> I removed the content-block class altogether, it was not needed. And
> page-content was being used in frappe.io site so thought of using it.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/frappe/frappe/pull/1744/files/eeb7da7c05a93167c4d779972de8a8e8daba35d1#r62218261>
>